c_src/device/cairo/cairo_fb.c

```diff
@@ -2,18 +2,21 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <linux/fb.h>
+#include <sched.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <time.h>
 
 #include "cairo_ctx.h"
 #include "comms.h"
 #include "device.h"
 #include "fontstash.h"
 #include "scenic_ops.h"
 
+#define FB0_TIMEOUT 60 //seconds
```

Comment:
```suggestion
#define FB0_TIMEOUT 60 /*seconds*/
```

I think we need to do this to prevent commenting out the rest of the line during macro expansion, no?

Reply:
Nevermind, the comment won't survive into the macro phase, we're good here!

Reply:
Yeah, comments get removed before the preprocessor phase. If there is some broken compiler that does it the other way around, then your solution would prevent it, but I think its ok as is.